delivery_auto_refresh/migrations/15.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+DELETE FROM ir_config_parameter
+WHERE ir_config_parameter.key LIKE '%delivery_auto_refresh%'
```

Comment:
- All former parameters started with `delivery_auto_refresh`...
- You don't need to specify the table in a query from a single table where there can be no ambiguity
```suggestion
WHERE key LIKE 'delivery_auto_refresh%'
```